tests/ui/autodiff/box.stderr

```diff
@@ -0,0 +1,56 @@
+analyzing function callee
+ + knowndata: ptr %X : {[-1]:Pointer} - {}
+ + retdata: {[-1]:Float@float}
+updating analysis of val: ptr %X current: {} new {[-1]:Pointer} from ptr %X Changed=1 legal=1
+updating analysis of val: ptr %X current: {[-1]:Pointer} new {[-1]:Pointer} from ptr %X Changed=0 legal=1
+updating analysis of val:   %X = fmul float %X, %X, !dbg !N current: {} new {[-1]:Float@float} from   %X = fmul float %X, %X, !dbg !N Changed=1 legal=1
+updating analysis of val:   %X = fmul float %X, %X, !dbg !N current: {[-1]:Float@float} new {[-1]:Float@float} from   %X = fmul float %X, %X, !dbg !N Changed=0 legal=1
+updating analysis of val: ptr %X current: {[-1]:Pointer} new {[-1]:Pointer} from   %X = load ptr, ptr %X, align 8, !dbg !N, !nonnull !14, !align !217, !noundef !N Changed=0 legal=1
+updating analysis of val:   %X = load ptr, ptr %X, align 8, !dbg !N, !nonnull !14, !align !217, !noundef !N current: {} new {} from   %X = load ptr, ptr %X, align 8, !dbg !N, !nonnull !14, !align !217, !noundef !N Changed=0 legal=1
```

Comment:
Can you please also add the `!nonnull !14` and the `!align !217`, since they also have metadata numbers? Try to not match the ones without `!`, like `align 4` since the alignment shouldn't change.